src/button-group/__integ__/button-group.test.ts

```diff
@@ -101,3 +104,30 @@ test(
     await expect(page.isFocused(copyButton.toSelector())).resolves.toBe(true);
   })
 );
+
+test(
+  'shows one tooltip at a time',
```

Comment:
Ensure there is only one tooltip. Test:

1. Click on copy with keyboard
2. Move focus to Remove - the tooltip is still on Copy and says "Copy"
3. Move focus to actions menu - now there are two tooltips shown